src/dataunion/DataUnion.ts

```diff
@@ -70,6 +70,20 @@ export type AmbMessageHash = string
 
 const log = debug('StreamrClient::DataUnion')
 
+const erc20AllowanceAbi = [{
```

Comment:
Move the ABI definition e.g. `to src/dataunion/abi.ts`?

Reply:
done